app/src/test/java/org/simple/clinic/sync/ModelSyncTest.kt

```diff
@@ -30,79 +34,85 @@ class ModelSyncTest {
   @get:Rule
   val rxErrorsRule = RxErrorsRule()
 
+  val syncConfigProvider = Single.fromCallable {
```

Comment:
Not important, but if the values aren't dynamic, you could just use `Single.just()`.

Reply:
I could. It's somehow ingrained into me to use `Single.fromCallable {}` for some reason. I'll change this when I have to work with this class next.

Reply:
Yea I know you enjoy using this function 😄